guides/metrics.md

```diff
@@ -7,6 +7,23 @@ Existing dashboards will need to be updated.
 It is important to know that metrics are created inside MongoosePush only when a certain event happens.
 This may mean that a freshly started MongoosePush node will not have all the possible metrics available yet.
 
+## Default dashboard
+
+MongoosePush 2.1.1 provides default Grafana dashboard where we can see some of the available metrics.
+You can create the dashboards using following command:
+
+```bash
+make dashboards
+```
+
+This starts and configures two containers:
+* `mongoose-push-grafana` - running Grafana, available at http://127.0.0.1:3000/
+* `mongoose-push-prometheus` - running Prometheus that is scraping the metrics from the `/metrics` endpoint, available at http://127.0.0.1:9090/
```

Comment:
Aren't those containers actually renamed to `mpush-xxx` instead of the longer `mongoose-push-xxx`?
```suggestion
* `mpush-grafana` - running Grafana, available at http://127.0.0.1:3000/
* `mpush-prometheus` - running Prometheus, which scraps the metrics from the `/metrics` endpoint, available at http://127.0.0.1:9090/
```